food-truck-api/src/main/java/food/truck/api/PositionConverter.java

```diff
@@ -0,0 +1,28 @@
+package food.truck.api;
+
+import lombok.extern.log4j.Log4j2;
+
+import javax.persistence.AttributeConverter;
+
+@Log4j2
+public class PositionConverter implements AttributeConverter<Position, String> {
+    @Override
+    public String convertToDatabaseColumn(Position l) {
```

Comment:
I changed RouteLocation so that it stores a Position instead of lat/lng doubles to make things easier when finding distance, etc. This is to make it save to the database correctly by converting the pair to a string and back.